src/app/helptext/system/certificates.ts

```diff
@@ -206,19 +207,22 @@ matches your certificate usage scenario.'),
     basic_constraints: {
       config: {
         placeholder: T('Basic Constraints Config'),
-        tooltip: T('The basic constraints extension identifies whether the \
- subject of the certificate is a CA and the maximum depth of valid \
- certification paths that include this certificate. <br> \
- See <a href="https://www.ietf.org/rfc/rfc3280.txt">RFC 3280, section 4.2.1.10</a> \
- for more information.'),
+        tooltip: T('Specify whether the certificate may be used for a Certificate Authority,\
+          and whether this extension is critical. Critical extensions must be recognized by the client or be rejected.\
+          Basic web certificates typically require CA disabled, and critical marked.'),
       },
       ca: {
         placeholder: T('CA'),
         tooltip: T('Identify this certificate as a Certificate Authority (CA).'),
       },
       enabled: {
         placeholder: T('Basic Constraints'),
-        tooltip: T('Activate this certificate extension.'),
+        tooltip: T('Activate the basic contstraints extension.\
```

Comment:
I recommend:

`Activate the Basic Constraints extension to identify whether the certificate's subject is a CA and the maximum depth of valid certification paths that include this certificate.`